### PR TITLE
Timebar Events

### PR DESCRIPTION
**Pull request by nerik**
Implements various Timebar modes as discussed with @satellitestudiodesign 

Screenshot 2020-05-10 at 18 16 39
Screenshot 2020-05-10 at 18 16 28
Screenshot 2020-05-10 at 18 16 10
Screenshot 2020-05-10 at 18 16 02
Screenshot 2020-05-10 at 18 15 27
Screenshot 2020-05-10 at 18 15 08

### CODE REVIEW

File: src/features/timebar/timebar.selectors.ts [hunk 1]
```diff
@@ -15,75 +18,93 @@ type TimebarTrack = {
...
-        tracksSegments.push({
-          segments: trackSegments,
-          color: config.color,
-        })
+      })
+      return {
+        color,
+        segmentsWithCurrentFeature,
+        // TODO Figure out this magic value
+        maxValue: 20,
```
Comment (j8seangel) [inline src/features/timebar/timebar.selectors.ts:80]: 🔮

Comment (j8seangel) [reply src/features/timebar/timebar.selectors.ts:80]: Where is this coming from? And why this value?

Comment (nerik) [reply src/features/timebar/timebar.selectors.ts:80]: The problem is that speed values returned by the API range from 0 to 1000+, when meaningful values should not really exceed 20/30 knots. Not sure yet what the best solution is:
- fix on the API side
- talk with Tim / Jaeyoon and establish a hardcoded value
- remove outliers mathematically (calculate for each value if they are within 1 standard deviation of mean?)

Comment (satellitestudiodesign) [reply src/features/timebar/timebar.selectors.ts:80]: - fixing it on the API means potentially breaking it for other use cases (unless is a url parameter)
- if we go the route of removing outliers based on standard deviation I'd choose 3 or 4 std to be safe
- hardcoded value seams fine for me (maybe change the comment to explain the domain knowledge behind the number?)

**Pull request merged**